lib/blueprinter/helpers/type_helpers.rb

```diff
@@ -3,13 +3,19 @@
 module Blueprinter
   module TypeHelpers
     private
-    def active_record_relation?(object)
-      !!(defined?(ActiveRecord::Relation) &&
-        object.is_a?(ActiveRecord::Relation))
+
+    def array_like_classes
+      [
+        Array,
+        defined?(ActiveRecord::Relation) && ActiveRecord::Relation,
+        *Blueprinter.configuration.array_like_classes
+      ].compact
     end
 
     def array_like?(object)
-      object.is_a?(Array) || active_record_relation?(object)
+      array_like_classes.any? do |klass|
```

Comment:
I got rid of the `array_like_class?` helper method since all this method did was call that method, but if you prefer it for readability I can move this logic back to that method and just call that method from here.